### ISSUE TITLE
Value returned by ‘initial-buffer-choice’ is not a live buffer: nil

### ISSUE THREAD

**Issue by ghost** [emacs-dashboard/emacs-dashboard#460]
dashboard-open works
but on startup I have 
Value returned by ‘initial-buffer-choice’ is not a live buffer: nil

**Comment by jcs090218** [comment]
Yeah, cuz the buffer isn't created before `dashboard-insert-startupify-lists` function being called. You can use `(get-buffer-create )` to avoid this.
